GitCommands/Git/GitModule.cs

```diff
@@ -3032,21 +3022,20 @@ public IReadOnlyList<string> GetAllBranchesWhichContainGivenCommit(ObjectId obje
                 return Array.Empty<string>();
             }
 
-            var output = _gitExecutable.GetOutput(
-                new GitArgumentBuilder("branch")
+            var args = new GitArgumentBuilder("branch")
                 {
                     { getRemote && getLocal, "-a" },
                     { getRemote && !getLocal, "-r" },
                     "--contains",
                     objectId
-                });
-
-            if (IsGitErrorMessage(output))
+                };
+            ExecutionResult exec = _gitExecutable.Execute(args);
+            if (!exec.ExitedSuccessfully)
             {
                 return Array.Empty<string>();
             }
```

Comment:
What do we do with the error message in this case?
After #9056, we should throw an `ExternalOperationException` here, which must be caught by the caller and be displayed in the UI element.

Reply:
adding comment